templates/galaxy/config/file_sources_conf.yml.j2

```diff
@@ -288,6 +288,12 @@
   public_name: ${user.preferences['zenodo_sandbox|public_name']}
   writable: true
 
+- type: inveniordm
+  id: berd_nfdi
+  doc: "A Single Place for Data in Business, Economics and Related Fields"
+  label: "BERD@NFDI InvenioRDM"
```

Comment:
@davelopez do you mean removing InvenioRDM?

Reply:
Yes, or calling it "BERD@NFDI Repository"? Up to you  :)

Reply:
Yes, thanks good suggestion!